first-party/ownership-extension/gradle/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+  id 'java-gradle-plugin'
+  id 'net.researchgate.release'
+  id 'maven-publish'
+  id 'eclipse'
+}
+
+dependencies {
+  compileOnly "com.google.cloud.tools:jib-gradle-plugin-extension-api:${dependencyVersions.GRADLE_EXTENSION}"
```

Comment:
ah since we provide this?

Reply:
Yeah... it's like cheating as we know Jib's implementation details. I don't expect the extension dev to do this but use `implementation` (and hope their specified Guava version doesn't conflict with ours). That said, I'm still unsure if we shouldn't say `implementation`.

Reply:
I'll merge this now to start kicking release, but feel free to bring this up later for discussion.